sdk/iot/modelsrepository/review/iot-modelsrepository.api.md

```diff
@@ -0,0 +1,51 @@
+## API Report File for "@azure/iot-modelsrepository"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { CommonClientOptions } from '@azure/core-client';
+import { OperationOptions } from '@azure/core-client';
+
+// @public
+export type dependencyResolutionType = "disabled" | "enabled" | "tryFromExpanded";
+
+// @public
+export interface GetModelsOptions extends OperationOptions {
+    dependencyResolution?: dependencyResolutionType;
+}
+
+// @public
+export function getModelUri(dtmi: string, repositoryUri: string, expanded?: boolean): string;
+
+// @public
+export function isValidDtmi(dtmi: string): boolean;
+
+// @public
+export class ModelError extends Error {
+    constructor(message: string);
+}
+
+// @public
+export class ModelsRepositoryClient {
+    constructor(options?: ModelsRepositoryClientOptions);
+    get apiVersion(): string;
+    get dependencyResolution(): dependencyResolutionType;
+    getModels(dtmis: string, options?: GetModelsOptions): Promise<{
+        [dtmi: string]: unknown;
```

Comment:
This is really generic! Is this intentional?

Reply:
yeah this is set as unknown because the structure is not fully known yet. I anticipate that before beta becomes GA this will be updated to a type.